packages/server/tests/integration/v1/roadmaps.spec.ts

```diff
@@ -12,7 +12,7 @@ describe("GET /api/v1/roadmaps", () => {
 
     expect(response.headers["content-type"]).toContain("application/json");
     expect(response.status).toBe(200);
-    expect(response.body.roadmaps).toHaveLength(0);
+    expect(response.body.data).toHaveLength(0); // Changed from 'roadmaps' to 'data'
```

Comment:
@kris70lesgo In reference to this [comment](https://github.com/logchimp/logchimp/pull/1017/files#r2249259432).